src/relay/backend/contrib/tensorrt/enable_tensorrt.cc

```diff
@@ -216,6 +216,18 @@ bool ConcatenateOpChecker(const CallNode* call, const std::string& op_name,
     LOG(INFO) << op_name << " not supported: cannot modify batch dimension.";
     return false;
   }
+  const auto tuple = call->args[0].as<TupleNode>();
+  if (tuple) {
```

Comment:
Generic suggestion:
This is an anti-pattern.

Instead
```suggestion
  if (!tuple) {
      // handle error??
      // return code
  }
  
  // get on with life here...
  
```